debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/SqlUtils.java

```diff
@@ -223,9 +223,16 @@ static String logMinerContentsQuery(OracleConnectorConfig connectorConfig, Strin
         StringBuilder query = new StringBuilder();
         query.append("SELECT SCN, SQL_REDO, OPERATION_CODE, TIMESTAMP, XID, CSF, TABLE_NAME, SEG_OWNER, OPERATION, USERNAME ");
         query.append("FROM ").append(LOGMNR_CONTENTS_VIEW).append(" ");
-        query.append("WHERE OPERATION_CODE IN (1,2,3,5) ");
-        query.append("AND SCN > ? ");
-        query.append("AND SCN < ? ");
+        query.append("WHERE ");
+        query.append("SCN > ? AND SCN <= ? ");
+        query.append("AND (");
+        // MISSING_SCN/DDL only when not performed by excluded users
+        query.append("(OPERATION_CODE IN (5,34) AND USERNAME NOT IN (").append(getExcludedUsers(logMinerUser)).append(")) ");
+        // COMMIT/ROLLBACK
+        query.append("OR (OPERATION_CODE IN (7,36)) ");
+        // INSERT/UPDATE/DELETE/DDL
```

Comment:
DDL is probably not part of this predicate so it should be removed form the comment